app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -148,7 +148,9 @@ def supports_migrate_for_all?(vms)
   end
 
   def version_higher_than?(version)
-    ems_version = api_version[/\d+\.\d+/x]
+    return false if api_version.nil?
```

Comment:
Could you add a test to manageiq-providers-ovirt/spec/models/manageiq/providers/redhat/infra_manager_spec.rb
`#context "#version_higher_than?" do` for a nil value ?